ch_tools/common/logging.py

```diff
@@ -0,0 +1,189 @@
+"""
+Logging module.
+"""
+
+import inspect
+import logging
+from typing import Any, Dict, Optional
+
+from loguru import logger
+
+this: Dict[str, Any] = {}
```

Comment:
1. Les's keep in this dict two state: the name and extra options. In that case we can pass directly `this[opt]`  to logger:
```
getLogger(this["module"]).opt(this['opt'])...
```

2. Let's rename like `current_logger' or 'logger_config'.